tests/test_google_sheets_datatypes.py

```diff
@@ -232,23 +232,21 @@ def test_run(self):
                             # verify the expected rows are actually Null
                             self.assertIsNone(value)
 
-                        elif value:
-
-                            # BUG_TDL-14369 | https://jira.talendforge.org/browse/TDL-14369
-                            #                 Skipping boolean column values becuase they do not correctly fall back to string
-                            if column == 'Boolean': # BUG_TDL-14369
-                                continue  # skip
+                        # As "'0" returns false which does not satisfy th below test case for boolean column
```

Comment:
Because the google-sheets API itself returns it with the discarding the `'` character in both the API calls i.e. with `formatted` as well as `unformatted `parameter.

Reply:
If it is coerced to a boolean then is it covered in the assertion on line 248?